sDownloads.ps1

```diff
@@ -0,0 +1,27 @@
+# Check directory sDownloads.ps1 runs from for sd-config file
+$ConfigPath = $PSScriptRoot + "\sd-config"
+
+# sd-config needs to stay with sDownloads.ps1
+# Change the path if you want sd-config somewhere else
+$Config = Import-Csv -Path $ConfigPath
+
+# Will work fine as long as you use the default Downloads folder
+$Downloads = "C:\Users\$env:UserName\Downloads\"
+
+$Files = $Downloads | Get-ChildItem
+$FileNames = $Files.Name
+
+ForEach($e in $Config) {
+    ForEach($f in $FileNames) {
+        $FilePath = $Downloads + $f
+        $FileExtension = [IO.Path]::GetExtension($f)
```

Comment:
These two items, you can populate from a file object directly. Pass $Files from above to `Get-Member` ($files | gm) and see what Properties are available to you there.

Reply:
In fact, these two vars may be completely redundant potentially.